src/test/e2e/42_values_test.go

```diff
@@ -0,0 +1,48 @@
+// SPDX-License-Identifier: Apache-2.0
+// SPDX-FileCopyrightText: 2021-Present The Zarf Authors
+
+// Package test provides e2e tests for Zarf.
+package test
+
+import (
+	"fmt"
+	"path/filepath"
+	"testing"
+
+	"github.com/stretchr/testify/require"
+)
+
+func TestValues(t *testing.T) {
+	t.Log("E2E: Values")
+
+	src := filepath.Join("src", "test", "packages", "42-values")
+	tmpdir := t.TempDir()
+
+	// Create the package
+	stdOut, stdErr, err := e2e.Zarf(t, "package", "create", src, "-o", tmpdir, "--skip-sbom", "--confirm", "--features=\"values=true\"")
+	require.NoError(t, err, stdOut, stdErr)
+
+	packageName := fmt.Sprintf("zarf-package-test-values-%s.tar.zst", e2e.Arch)
+	path := filepath.Join(tmpdir, packageName)
+
+	// Deploy the package
+	stdOut, stdErr, err = e2e.Zarf(t, "package", "deploy", path, "--confirm")
```

Comment:
I think it would be good to test `--values` here. One of the harder things to unit test since it's at the cmd level and deploy.